### DISCUSSION TITLE
Search box in filters

### DISCUSSION THREAD

**Discussion by brian6932** [Chatterino/chatterino2#5741] [Ideas] [answered]
I have a lot of filters, I'd like to search by their keys, thank you :P

**Comment by Coldblackice** [comment]
Would also love this ability, especially those that have a lot of filters to sort/scroll through.

**Reply by Nerixyz** [reply]
There's a PR to allow the search to filter filters in https://github.com/Chatterino/chatterino2/pull/5890.

**Comment by pajlada** [comment, marked as answer]
This was implemented in https://github.com/Chatterino/chatterino2/pull/5890 and will be available in the next release
